src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/ApprenticeshipStopDateChangedEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public class ApprenticeshipStopDateChangedEvent
```

Comment:
Naming events so that they have parity with the commands can be a nice convention i.e. `UpdatedApprenticeshipStopDateEvent` could be published as a result of handling `UpdateApprenticeshipStopDateCommand`. What do you think?

Reply:
I don't tend to like naming events after command handlers (but I see the advantages). These messages will be raised by V2 events in the near future, so to tie the name to the existing 'old' event won't make sense in the future